endpoints/get.go

```diff
@@ -38,9 +46,16 @@ func NewGetHandler(storage backends.Backend, metrics *metrics.Metrics, allowCust
 
 func (e *GetHandler) handle(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
 	e.metrics.RecordGetTotal()
+
+	if utils.RandomPick(e.cfg.refererLogRate) == true {
```

Comment:
The `== true` check isn't required.

Reply:
Removed

Reply:
Still looks like it's there. Did you forget to save or commit?

Reply:
I don't know 😅 . Corrected